php/config.php

```diff
@@ -12,4 +12,5 @@
   define('DB_DATABASE', 'board18');
   define('DB_USER', 'board18');
   define('DB_PASSWORD', 'board18');
-?>
+  define('ENABLE_REGISTRATION', true);
+?>
```

Comment:
This change is really important as it affects the response from ajax calls. Adding a newline here will result in `\nsuccess` rather than the required `success`